\documentclass[11pt,oneside,letterpaper]{article}

% graphicx package, useful for including eps and pdf graphics
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{subfig}
%\DeclareGraphicsExtensions{.pdf,.png,.jpg}

% basic packages
\usepackage{color}
\usepackage{parskip}
\usepackage{float}
\usepackage{url}
\urlstyle{same}


% reference figures across documents
\usepackage{xr}
\externaldocument{mers-structure_supp}

% text layout
\usepackage{geometry}
\geometry{textwidth=15cm} % 15.25cm for single-space, 16.25cm for double-space
\geometry{textheight=22cm} % 22cm for single-space, 22.5cm for double-space

% helps to keep figures from being orphaned on a page by themselves
\renewcommand{\topfraction}{0.85}
\renewcommand{\textfraction}{0.1}

% bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,font=small]{caption}

% review layout with double-spacing
%\usepackage{setspace}
%\doublespacing
%\captionsetup{labelfont=bf,labelsep=period,font=doublespacing}

% cite package, to clean up citations in the main text. Do not remove.
%\usepackage{cite}
\usepackage{natbib}
%\renewcommand\citepleft{(}
%\renewcommand\citepright{)}
%\renewcommand\citepform[1]{\textsl{#1}}

\usepackage{amsmath}

% Remove brackets from numbering in list of References
%\renewcommand\refname{\large References}
%\makeatletter
%\renewcommand{\@biblabel}[1]{\quad#1.}
%\makeatother

\usepackage{authblk}
\renewcommand\Authands{ \& }
\renewcommand\Authfont{\normalsize \bf}
\renewcommand\Affilfont{\small \normalfont}
\makeatletter
\renewcommand\AB@affilsepx{, \protect\Affilfont}
\makeatother

% comments
\usepackage{ulem}
\definecolor{purple}{rgb}{0.459,0.109,0.538}
\def\tb#1#2{\sout{#1} \textcolor{purple}{#2}}
\def\tbc#1{\textcolor{purple}{[#1]}}
\def\gdc#1{\textcolor{blue}{[#1]}}
\def\lmc#1{\textcolor{green}{[#1]}}

% symbols
\newcommand{\chiSq}{\chi^{2}_{df}} %LM: ancient DNA? =P
\newcommand{\dtmrca}{\Delta_\mathrm{TMRCA}}
\newcommand{\undtmrca}{\delta_\mathrm{TMRCA}}
\newcommand{\dspr}{d_\mathrm{SPR}}

%%% TITLE %%%
\title{\vspace{1.0cm} \LARGE \bf Structured population models demystify MERS-CoV epidemiology}

\author[1]{Gytis Dudas}
\author[2]{Luiz Max Carvalho}
\author[2,3,4]{Andrew Rambaut}
\author[1]{Trevor Bedford}

\affil[1]{Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA}
\affil[2]{Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK}
\affil[3]{Fogarty International Center, National Institutes of Health, Bethesda, MD, USA}
\affil[4]{Centre for Immunology, Infection and Evolution at the University of Edinburgh, Edinburgh, UK}

\date{\today}

\begin{document}
\maketitle

\begin{abstract}

Middle East Respiratory Syndrome coronavirus (MERS-CoV) is a zoonotic virus originating in camels that has been causing significant mortality and morbidity in humans in the Arabian Peninsula.
The epidemiology of the virus remains poorly understood, with hospital outbreaks, isolated cases with known exposure to camels and apparent community transmission occurring simultaneously.
Whilst traditional and seroepidemiological studies have been employed extensively throughout the epidemic, viral sequence data have not been utilised to their full potential in understanding transmission patterns within the outbreak.
Here we use existing MERS-CoV sequence data to explore the phylodynamics of the virus in two of its known major hosts, humans and camels.
We employ structured coalescent models to show that long-term MERS-CoV evolution occurs exclusively in camels, whereas humans act as a transient, and ultimately terminal host. 
\tbc{Fields' Virology textbook defines `terminal host' as `one that is not involved in maintaining the virus, such infections rarely result in transmission to other human or non-human hosts'. I think you want a different word here. Maybe `transient host'?}
\gdc{Good point about the term. Hope the definition leans towards `one not involved in maintenance' rather than `limited to index case'. Asked a friend working on zoonoses about this.}
By analysing the distribution of human outbreak cluster sizes and zoonotic introduction times we show that MERS-CoV is unlikely to become endemic in humans and that human outbreaks in the Arabian peninsula for the most part are driven by seasonally varying zoonotic transfer of viruses from camels.
% \tbc{Powerful and succinct abstract. Great.}

\end{abstract}

\pagebreak

\section*{Introduction}
%% Middle East Respiratory Syndrome Coronavirus (MERS-CoV) is a zoonotic infection of humans in the Arabian Peninsula originating from camels. % AR - a virus isn't an infection and not all infections in the Arabian Peninsula
Middle East Respiratory Syndrome Coronavirus (MERS-CoV), endemic in camels in the Arabian Peninsula, is the causative agent of zoonotic infections and limited outbreaks in humans. % ? something like this ?
The virus, first discovered in 2012 \citep{zaki_isolation_2012,boheemen_genomic_2012}, has caused more than 2000  infections and over 700 deaths, according to the World Health Organization (WHO) \citep{mersfactsheet}.
Its epidemiology remains obscure, largely because outbreaks are observed among the most severely affected individuals, such as older males with comorbidities \citep{assiri_2013,group_state_2013}.
Whilst contact with camels is often reported, patients may not recall contact with \lmc{other?} livestock, suggesting an unobserved community contribution to the outbreak \citep{group_state_2013}.
Studies into MERS-CoV epidemiology in the past often relied on serology to identify factors associated with MERS-CoV exposure in potential risk groups \citep{reusken_occupational_2015,reusken_2013}.
Indeed, such data have been used to show high seroprevalence in camels \citep{muller_2014,corman_antibodies_2014,chu_2014,reusken_2013,reusken_2014} and evidence of contact with MERS-CoV in workers with occupational exposure to camels \citep{reusken_occupational_2015,muller_presence_2015}.
Epidemiological modeling approaches have been used to look at traditional data sources of case clusters through time, space and across hosts \citep{cauchemez_unraveling_2016}.


Although traditional epidemiological approaches yield important clues about exposure patterns and potential for larger outbreaks, much inevitably remains opaque to such approaches \lmc{why, specifically?}.
The new era of genomic epidemiology, however, has repeatedly shown the utility of viral sequence data in outbreak scenarios \citep{quick_real-time_2016,arias_rapid_2016}, where data are relatively cheap to produce.
These data can stand in for diagnostics and often yield a highly detailed picture of an epidemic when complete genome sequencing is performed consistently and appropriate metadata collected \citep{dudas_virus_2017}.
Often only sequence data can \lmc{help} pinpoint sources of pathogens and discriminate between multiple and single source scenarios \citep{gire_genomic_2014}, which are fundamental to quantifying risk \citep{grubaugh_multiple_2017}.
Sequencing MERS-CoV has been performed as part of initial attempts to link human infections with the camel reservoir \citep{memish_human_2014}, nosocomial outbreak investigations \citep{assiri_hospital_2013} and routine surveillance \citep{park_acute_2015}.
A large portion of MERS-CoV sequences come from hospital outbreaks \lmc{when you say hospital outbreak, do you mean an actual outbreak taking place inside the hospital?}, where sequence data have been used to determine whether infections were isolated introductions or were part of a larger hospital-associated outbreak \citep{fagbo_molecular_2015}.
Similar studies on MERS-CoV have taken place at broader geographic scales, such as cities \citep{cotten_2013}.


It is widely accepted that recorded human MERS-CoV infections are a result of at least several introductions of the virus into humans \citep{cotten_2013} and that contact with camels is a major risk factor for developing MERS, per WHO guidelines \lmc{how about a citation here?}.
Previous studies attempting to quantify the actual number of spillover infections have either relied on traditional, and necessarily limited, approaches \citep{cauchemez_unraveling_2016} or employed methods agnostic to signals of population structure within sequence data \citep{zhang_evolutionary_2016}.
Here we use existing MERS-CoV sequence data to investigate the population structure of the virus between two of its known hosts, humans and camels by using an explicit model of metapopulation structure and migration between discrete sub-populations, referred to here as demes \citep{vaughan_efficient_2014}.

%Unlike discrete trait analyses (DTA, or ``mugration'' models) %% AR - I don't like 'DTA' as name for this model - it is not specific enough. The model is a continuous time Markov process
%that treat a potential trait of interest (such as host state) as if it were a nucleotide site with an arbitrary number of states, population structure models explicitly incorporate distinct sampling patterns and population dynamics across demes.
Unlike approaches that model the host species as a discrete trait of the virus using continuous-time Markov processes (or simpler, parsimony based, approaches) [cite here, Philippe's bat rabies and Sam's avian flu?, others?], population structure models explicitly incorporate the distinct sampling patterns, population dynamics within demes and the migration between demes.
By estimating independent coalescence rates for MERS-CoV in humans and camels, as well as migration patterns between the two demes, we show that long-term viral evolution of MERS-CoV occurs exclusively in camels.
Our results suggest that spillover events into humans are seasonal and might be associated with the calving season in camels.
Once human MERS-CoV infections are established, however, we find that MERS-CoV is poor at transmitting between humans.
Using Monte Carlo simulations we show that $R_{0}$ for MERS-CoV circulating in humans is much lower than the epidemic threshold of 1.0 and that correspondingly the virus has been introduced into humans hundreds of times.

%\begin{figure}[h]
% \centering
%	\includegraphics[width=0.45\textwidth]{figures/}
%	\caption{\textbf{}
%
%	}
%	\label{}
%\end{figure}

\section*{Results}

\subsection*{MERS-CoV is predominantly a camel virus}

The structured coalescent approach we employ \citep{vaughan_efficient_2014} identifies camels as a reservoir host where most of MERS-CoV evolution takes place (Figure \ref{mcc}), whilst human MERS outbreaks are transient and terminal with respect to long-term evolution of the virus (Figure~\ref{exploded}).
Across 174 MERS-CoV genomes collected from humans, we detect a median of 56 individual camel-to-human cross-species transmissions (95\% highest posterior density (HPD): 48--63).
Whilst we identify \lmc{estimate?} a median of 3 (95\% HPD: 0--12) human-to-camel migrations, the 95\% HPD interval includes zero and we find that no such migrations are found in the maximum clade credibility tree (Figure~\ref{mcc}).
We conclude that most human-to-camel migrations are induced by the migration rate prior, whilst some are derived from phylogenetic proximity of human sequences to the apparent ``backbone'' of the phylogenetic tree.
It is most apparent in the lineages existing in early-mid 2013 that lead up to sequences comprising the MERS-CoV clade dominant in 2015, where poor sampling of MERS-CoV genetic diversity from camels allows \lmc{is ``allow'' the best word here? I understand you' rather not use ``biases'' or ``misleads'', though...} the model to consider humans as a potential alternative host.
The first sequences of MERS-CoV from camels do not appear in our data until November 2013.
Our finding of negligible human-to-camel transmission is robust to choice of prior (Figure~\ref{prior}).

The repeated and asymmetric introductions of short-lived clusters of MERS-CoV sequences from camels into humans leads us to conclude that MERS-CoV epidemiology in humans is dominated by zoonotic transmission (Figure~\ref{mcc} and~\ref{exploded}).
We observe dense terminal clusters of MERS-CoV circulating in humans that are of no subsequent relevance to the evolution of the virus.
These clusters of presumed human-to-human transmission are then embedded within extensive diversity of MERS-CoV lineages inferred to be circulating in camels, a classic pattern of source-sink dynamics.
Our analyses recover these results despite sequence data heavily skewed towards non-uniformly \lmc{maybe ``non-uniformly'' is redundant here} sampled human cases and are robust to choice of prior.
We therefore argue that instances of human infection with MERS-CoV are more common than currently thought, with exceedingly short transmission chains mostly limited to index cases that might be mild and ultimately not detected by surveillance or sequencing.

\begin{figure}[h]
 \centering
	\includegraphics[width=0.65\textwidth]{figures/mers_mcc.png}
	\caption{\textbf{Typed maximum clade credibility tree of MERS-CoV genomes from humans and camels.}
	Maximum clade credibility (MCC) tree showing inferred ancestral hosts for MERS-CoV recovered with the structured coalescent.
	Vast majority of MERS-CoV evolution is inferred to occur in camels (orange) with human outbreaks (blue) representing evolutionary dead-ends for the virus.
  Confidence in host assignment is depicted as a colour gradient, with increased uncertainty in host assignment (posterior probabilities close to 0.5) shown as grey.
  Some of the branches leading up to zoonotic transmission of MERS-CoV into humans are too long to have spent much of their time in humans, given our \textit{a priori} belief that MERS-CoV cannot circulate in humans for long periods of time.
  Thus branches encompassing cross-species transmissions are depicted as being in camels right up to the common ancestor or tip of each human outbreak.
	Whilst large clusters of human cases are apparent in the tree, significant contributions to human outbreaks are made by singleton sequences, likely representing recent cross-species transmissions that were caught early.
	}
	\label{mcc}
\end{figure}



\subsection*{Signals of MERS-CoV seasonality}
Unlike the most basic \lmc{simpler?} ``mugration'' approaches that simply infer and label the internal nodes of a phylogeny, structured population models possess the ability to estimate migration times.
We use the posterior distribution of MERS-CoV introduction times for human clades observed in the MCC tree (Figure \ref{mcc}) to model seasonal variation in zoonotic transfer of viruses.
We identify four months (April, May, June, July) when the odds of MERS-CoV introductions are increased (Figure \ref{seasonality}) and four when the odds are decreased (August, September, November, December).
Camel calving is reported to occur from October to February \citep{almutairi_non-genetic_2010}, with rapidly declining maternal antibody levels in calves within the first weeks after birth \citep{wernery_camelid_2001}.
It is possible that MERS-CoV sweeps through each new camel generation once critical mass of susceptibles is reached, leading to a sharp rise in incidence of the virus in camels resulting in increased force of infection sparking human outbreaks.
Strong influx of susceptibles and subsequent sweeping outbreaks in camels may explain evidence of widespread exposure to MERS-CoV in camels from seroepidemiology \citep{muller_2014,corman_antibodies_2014,chu_2014,reusken_2013,reusken_2014}.

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_seasonality.png}
	\caption{\textbf{MERS-CoV seasonality.}
A) Violin plots coloured by month showing the 95\% highest posterior density interval for odds ratios of MERS-CoV introductions into human populations.
Posterior means are indicated with circles.
Evidence for increased or decreased risk (95\% HPD excludes 1.0) for introductions are indicated by black or white circles, respectively.
Grey area spanning October to February indicates the camel calving season.
B) Each clade entering humans is positioned based on the median introduction time, as recovered by structured coalescent analyses and coloured by time of year with height indicating number of descendent sequences recovered from human cases.
95\% highest posterior density intervals for zoonotic introduction of MERS-CoV into human populations observed in the MCC tree (Figure \ref{mcc}) are indicated with coloured lines, coloured by median estimated introduction time.
The black dotted line indicates the joint probability density for introductions.
Grey areas spanning October to February indicate the reported camel calving season.\lmc{if these two panels indeed end up being presented together, this is superfluous}
	}
	\label{seasonality}
\end{figure}

%\begin{figure}[h]
% \centering
%	\includegraphics[width=0.45\textwidth]{figures/}
%	\caption{\textbf{}
%
%	}
%	\label{}
%\end{figure}

\subsection*{MERS-CoV is poorly suited for human transmission}

Structured coalescent approaches clearly show humans to be a terminal host %\tbc{again, I don't know if this is the right term or not}
for MERS-CoV, implying poor transmissibility.
However, we wanted to translate this observation into the basic reproductive number, $R_{0}$, which is more familiar to epidemiologists and provides expectations\lmc{insights?} of epidemic behavior.
The parameter $R_{0}$, along with the degree of heterogeneity in tranmission quantified by a dispersion parameter $\omega$, can be used to determine the distribution of total cases that result from a single introduction event into the human population (Equation~\ref{clusterLikelihood}).
We estimate $R_{0}$ along with other relevant parameters via Monte Carlo simulation in two steps.
First, we simulate case counts across multiple outbreaks totalling 2000 cases using Equation \ref{clusterLikelihood} with a fixed dispersion parameter $\omega$ and then subsample using a multivariate hypergeometric distribution from each case cluster to simulate sequencing of a fraction of cases.
Sequencing simulations take place at different levels of bias, wherein bias enriches sequencing of larger case clusters.
This is a particularly pressing issue, since \textit{a priori} we expect large hospital outbreaks of MERS to be overrepresented in sequence data, whereas sequences from index cases will be sampled exceedingly rarely.
We record the $R_{0}$ values, as well as the number of case clusters (equivalent to number of zoonotic introductions) for simulations that produce sequence cluster sizes with mean, median and standard deviation falling within the 95\% highest posterior density for these summary statistics observed in MERS-CoV data from structured coalescent analyses (left-hand panels in Figure \ref{ABC}).

We find that observed phylogenetic patterns of sequence clustering strongly support $R_{0}$ below 1.0 (middle panels in Figure \ref{ABC}).
For increasing levels of bias mean $R_{0}$ values observed in matching simulations are 0.844, 0.730, and 0.683, respectively.
Whilst the 95\% percentiles for $R_{0}$ values are close to 1.0 (0.720--0.995) for the unbiased sequencing simulation (\textit{i.e.} uniform sequencing efforts, in which every case is equally likely to be sequenced), we also note that increasing levels of bias are considerably better at generating MERS-CoV-like sequence clusters.
Under unbiased sequencing only 0.6\% (2282 out of 363 000) of simulations fit our phylogenetic observations, while 2.7\% and 2.7\%\lmc{again?} of simulations fit for bias levels of 2.0 and 3.0, respectively.
This is apparent from the heights of the histograms in Figure \ref{ABC}, which indicate a higher marginal likelihood for bias levels of 2.0 and 3.0 relative to bias of 1.0 (10\% support for bias level 1.0, 45\% support for bias level 2.0, and 45\% support for bias level 3.0).\lmc{I don't follow this argument...}
Simulations with bias suggest that $R_{0}$ values could have ranges [0.605--0.870] (bias = 2) and [0.555--0.825] (bias = 3).

Lower values for $R_{0}$ in turn suggest relatively large numbers of zoonotic transfers of viruses into humans (right-hand panels in Figure \ref{ABC}).
The median number of cross-species introductions observed in simulations matching MERS data without bias are 339 (95\% percentiles 255--431).
These numbers jump up to 558 (95\% percentiles 424--720) for bias = 2 and 650 (95\% percentiles 481--850) for bias = 3 simulations, which as mentioned previously match considerably better to MERS phylogenetic data.
Our results suggest a large number of unobserved MERS index cases.\lmc{does the definition of ``index case'' allow for ``multiple''?}
Given this, we also performed simulations where the total number of cases is doubled to 4000 to explore the impact of dramatic underestimation of MERS cases.
In those sets of analyses $R_{0}$ values tend to decrease even further as numbers of introductions go up, although very few simulations match currently observed MERS-CoV sequence data (Figure \ref{extra_cases}).

Overall, our analyses indicate that MERS-CoV is poorly suited for human-to-human transmission, with an estimated $R_{0}<1.0$ and sequence sampling likely to be biased towards large hospital outbreaks.
% \tbc{I didn't understand what you meant by `sequence sampling wholly unrepresentative of viral diversity found in humans'.}
Given these findings, and the fact that large outbreaks of MERS occurred in hospitals, the combination of frequent spillover of MERS-CoV into humans and occasional outbreak amplification via poor hygiene practices \citep{assiri_hospital_2013,chen_comparative_2017} appear sufficient to explain observed epidemiological patterns of MERS-CoV.
%\tbc{It would be really nice to be able to reference something here on hospital spread of MERS (or SARS) owing to hygiene practices.}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_epi.png}
	\caption{\textbf{Results of Monte Carlo simulations.}
Each row corresponds to a different bias value used to concentrate the multivariate hypergeometric distribution used to simulate sequencing, and goes from 1 (no bias) to 2, and 3 (increasing levels of bias \lmc{make sequences from large clusters more likely to be picked} ).
Leftmost scatter plots show a 10\% subsample of all outbreak and sequencing simulations, coloured by the $R_{0}$ value used for the simulation.
The dotted rectangle identifies the 95\% highest posterior density bounds for sequence cluster size mean and standard deviation observed for MERS-CoV data.
The distribution of $R_{0}$ values found within the dotted rectangle is shown in the middle, on the same y axis across all levels of bias.
Bins falling inside the 95\% percentiles are coloured by $R_{0}$, as in the leftmost scatter plot.
The distribution of total number of introductions associated with simulations matching MERS-CoV sequence clusters is shown in the plots on the right, on the same y axis across all levels of bias.
Darker shade of grey indicates bins falling within the 95\% percentiles.
These Monte Carlo simulations indicate $R_{0}$ for MERS-CoV is likely to be below 1.0, with biased sequencing and numbers of zoonotic transmissions numbering in the hundreds.
% \tbc{Not at all essential, but I can imagine that an initial panel A showing the logic of the two step ABC process would do a whole lot to alleviate confusion. I'm picturing a set of tables numbered with case counts in a top row and then a bottom row with sequence counts. This could also work better as a separate figure referenced from the Methods.}
	}
	\label{ABC}
\end{figure}


%\begin{figure}[h]
% \centering
%	\includegraphics[width=0.65\textwidth]{figures/}
%	\caption{\textbf{}
%
%	}
%	\label{}
%\end{figure}



\subsection*{Recombination shapes MERS-CoV diversity}
Recombination has been shown to occur in all genera of coronaviruses, including MERS-CoV \citep{lai_1985,makino_1986,keck_1988,kottier_1995,herrewegh_1998}. %\tbc{needs citations}.
In order to explore the role of recombination in shaping MERS-CoV genetic diversity we used two recombination detection approaches across partitions of taxa corresponding to inferred MERS-CoV clades.
Both methods rely on sampling parental and recombinant alleles within the alignment, although each quantifies different signals of recombination.
One hallmark of recombination is the ability to carry alleles derived via mutation from one lineage to another, that appear as repeated mutations taking place in the recipient lineage, somewhere else in the tree.
PHI (pairwise homoplasy index) test quantifies the appearance of these excessive repeat mutations (homoplasies) within an alignment \citep{bruen_simple_2006}.
Another hallmark of recombination is spatial clustering of alleles in the genome, due to how template switching, the primary mechanism of recombination in RNA viruses, occurs.
3Seq relies on the spatial structure of nucleotide similarities between sequence triplets -- two potential parent-donors and one candidate offspring-recipient sequences \citep{boni_exact_2007}.

Both tests can give spurious results in cases of extreme rate heterogeneity and sampling over time \citep{dudas_mers-cov_2016}, but both tests have not been reported to fail simultaneously.
PHI and 3Seq methods consistently identify most of the apparent `backbone' of the MERS-CoV phylogeny as encompassing sequences with evidence of recombination (Figure \ref{recombination_tree}).
Neither method can identify where in the tree recombination occurred, but each full asterisk in Figure \ref{recombination_tree} should be interpreted as the minimum partition of data that still captures both donor and recipient alleles involved in a recombination event.
This suggests a non-negligible contribution of recombination in shaping existing MERS-CoV diversity.
As done previously \citep{dudas_mers-cov_2016}, we show large numbers of homoplasies in MERS-CoV data (Figure \ref{incompatibilities}) with some evidence of spatial clustering of such alleles.
Homoplasies present in multiple strains at a time, indicating ancestral events that have been successful.
Although the proportion of time that the virus is inferred to spend in camels alone may be sufficient to argue that camels are the host where MERS-CoV recombines, incidence of MERS-CoV is known to be much higher in camels \citep{muller_2014,corman_antibodies_2014,chu_2014,reusken_2014,ali_systematic_2017}.
This provides ideal conditions for co-infection, which is a pre-requisite for RNA virus recombination to occur\lmc{stupid biostatistician here: is it?}.

Conversely, our results strongly suggest that co-infection of humans with distinct lineages of MERS-CoV should be exceedingly rare.
We find little evidence that recombination might be interfering with human outbreak cluster inference beyond what could be expected of phylogenetic noise (Figure \ref{recombination}A).
And whilst we observe evidence of distinct phylogenetic trees from different parts of the MERS-CoV genome (Figure \ref{recombination}B), human clades are minimally affected and large portions of the posterior probability density in both parts of the genome are concentrated in shared clades (Figure \ref{flower}).
Observed departures from strictly clonal evolution suggest that whilst recombination is a serious issue in MERS-CoV phylogenies, its effects on the human side of MERS outbreaks is minimal, as expected.
Overall we advise extreme caution when interpreting branches in MERS-CoV phylogenies that are suspected of having spent any time in camels, especially when attempting to identify origins of human cases.
On the other hand, the largely clonal evolution of MERS-CoV in humans, despite limited diversity, should be more amenable to traditional phylogenetic approaches.


% \begin{figure}[h]
%  \centering
% 	\includegraphics[width=0.65\textwidth]{figures/mers_chain.png}
%   \includegraphics[width=0.65\textwidth]{figures/mers_fragments.png}
% 	\caption{\textbf{Tangled chain of MERS-CoV genome and its parts.}
% 	Tangled chain of the full genome (a), all nucleotides up to position 21000 (b) and all nucleotides past position 21000 (c) of MERS-CoV.
% 	The same taxa, if available in neighbouring trees, are connected via coloured lines (human sequences in blue, camel sequences in red).
% 	Tree branches are coloured by inferred ancestral host state (human in grey, camel in black).
% 	Whilst some of apparent incongruities are caused by having less data in some of the fragments, inconsistencies between topologies occur across internal branches inferred to be in the camel host.
% 	Human clusters in grey change phylogenetic positions between the trees wholesale, with minor incongruences within clusters.
% 	This is evidence for recombinant viruses generated in the reservoir entering human populations.
% 	Coding sequences in the MERS-CoV genome are indicated with arrows at the bottom of the plot with position 21000 indicated by the black line.}
% 	\label{mers_chain}
% \end{figure}

% \begin{figure}[h]
% \centering
%
% 	\caption{\textbf{Pairwise co-occurence matrix for human clusters.}
% Heatmap shows the posterior probability that a pair of tips from trees of genomic fragments fall within the same clade - tips from fragment 1 are on the x axis, tips from fragment 2 are on the y axis.
% Tips are ordered by their appearance in tree of genome fragment 2 (positions from nucleotide 21000 onwards) reduced to just the human tips and coloured by inferred host (blue for human, orange for camel) on the left.
% Human clusters are largely well-supported as monophyletic and consistent across trees of both genomic fragments.
% 	}
% 	\label{coocurrence}
% \end{figure}
\begin{figure}%
    \centering
    \subfloat{{\includegraphics[width=7cm]{figures/mers_fragments.png} }}%
    \qquad
    \subfloat{{\includegraphics[width=5cm]{figures/mers_chain.png} }}%
    \caption{\textbf{Recombinant features of MERS-CoV phylogenies}.
    A) Heatmap shows the posterior probability that a pair of tips from trees of genomic fragments fall within the same clade - tips from fragment 1 are on the x axis, tips from fragment 2 are on the y axis.
    Tips are ordered by their appearance in tree of genome fragment 2 (positions from nucleotide 21000 onwards) reduced to just the human tips and coloured by inferred host (blue for human, orange for camel) on the left.
    Human clusters are largely well-supported as monophyletic and consistent across trees of both genomic fragments.
    B) Phylogenies derived from MERS-CoV genome nucleotides up to position 21000 (left) and all nucleotides past position 21000 (right), reduced to just the human tips.
  	Same taxa are connected via coloured lines indicating their vertical position in the tree on the right.
  	Branches are coloured by inferred ancestral host state (human in blue, camel in orange).
  	Whilst some of apparent incongruities are caused by having less data in some of the fragments, inconsistencies between topologies occur across internal branches inferred to be in camels.\lmc{I'm sure this is important, but I didn't get why. Referees might not either...}
  	Human clusters in blue change phylogenetic positions between the trees wholesale\lmc{wholesale??}, with minor incongruences within clusters.
  	This is evidence for recombinant viruses generated in the reservoir\lmc{(camels)} entering human populations.
    }%
    \label{recombination}%
\end{figure}

\subsection*{Demographic trajectory of MERS-CoV in camels}
Since MERS-CoV in humans evolves under a distinct demographic regime, we split our sequence dataset based on whether sequences were collected in humans or camels.
We supplement camel sequence data with a single earliest sequence from each human cluster, treating viral diversity present in humans as a sentinel sample of MERS-CoV diversity circulating in camels.
This removes conflicting demographic signals sampled during human outbreaks, where densely sampled closely related sequences from humans could be misconstrued as evidence of demographic crash in the viral population.

Our attempt at controlling the confounding effects of different sampling regimes between humans and camels revealed a troubling pattern, however.
We find that this camel-like dataset results in poor MCMC performance, where independent chains reach stationary distributions which are not necessarily the same.
Out of the five runs we set up two chains converged to one stationary distribution, two to another and the last one to a third.
Demographic trajectories recovered by the two main stationary distributions are very similar and differences between the two appear to be caused by convergence onto different tree topologies, with onward effects on evolutionary rate estimates and distributions of coalescence times.
This non-convergence effect may have been masked previously by the use of all available MERS-CoV sequences from humans which may have lead MCMC towards one of the multiple stationary distributions.

Despite lack of convergence, neither of the two demographic reconstructions show evidence of fluctuations in the relative genetic diversity (N$_{e}\tau$) of  MERS-CoV over time (Figure \ref{skygrid}).
However, we do note that estimates of relative genetic diversity are relatively low overall, and MERS-CoV phylogeny resembles a ladder often seen in human influenza A virus phylogenies \citep{bedford_strength_2011}.
This raises the possibility that MERS-CoV populations undergo lineage turnover at a rate not often seen in other livestock viruses (\textit{e.g.} swine influenza A viruses \citep{vijaykrishna_long-term_2011}).
Part of this could be caused by MERS-CoV ability to re-infect camels previously exposed to the virus \citep{ali_systematic_2017}, which may result in a process akin to antigenic drift, where the depletion of susceptible calves leads to ever increasing selective pressure for the virus to evolve antigenically.
Whilst this could be tested by looking for seasonally varying patterns in selection on MERS-CoV surface proteins, recombination would remain a key confounding problem.

% In order to disentangle distinct demographic modes within the phylogeny of MERS-CoV we split our alignment according to whether the virus was collected from camels or humans.
% For sequences collected from humans and inferred to be in the same cluster we only keep the earliest sequence, and arrive at a sequence dataset that for the most part is comprised of viral diversity present in camels, and supplemented with human sequences that are treated as sentinel samples.


\section*{Discussion}

\subsection*{Role of population structure in phylodynamics}
\lmc{Detecting?} Population structure yields important insight into any biological system.
It also often implies the existence of barriers experienced by an organism, be it geographic \citep{dudas_virus_2017}, ecological \citep{smith_dating_2009} or evolutionary \citep{turner_genomic_2005,dudas_reassortment_2015}.
Understanding what barriers exist and how they affect transmission of viruses remains one of the key parameters in designing efficient interventions.
In this study we aimed to understand the drivers of MERS coronavirus transmission in humans and what role the camel reservoir plays in perpetuating the epidemic in the Arabian peninsula.
Whilst there are clear indications that human cases of MERS are largely the result of sporadic spillover from camels, sequence data have not yet been fully utilised in estimating basic epidemiological parameters, such as R$_{0}$ or the number of zoonotic introductions of MERS-CoV into humans.

We showed that currently existing models of population structure \citep{vaughan_efficient_2014} can identify distinct demographic modes in MERS-CoV data, where viruses continuously circulating in camels repeatedly jump into humans and cause small outbreaks doomed to extinction (Figures \ref{mcc}, \ref{exploded}).
This inference succeeds under different choices of priors for unknown demographic parameters (Figure \ref{prior}) and in the presence of strong biases in sequence sampling schemes (Figure \ref{ABC}).
From sequence data we identify at least 50 zoonotic introductions of MERS-CoV into humans from the reservoir (Figure \ref{mcc}), from which we extrapolate that hundreds more such introductions must have taken place (Figure \ref{ABC}).
We also looked at potential seasonality in MERS-CoV spillover into humans, which is an additional advantage of explicitly modeling population structure.
Our analyses indicated a period of three months where the odds of a sequenced spillover event are increased, with timing consistent with an enzootic amongst camel calves (Figure \ref{seasonality}).
As a result of our identification of large and asymmetric flow of viral lineages into humans we also find that the basic reproduction number for MERS-CoV in humans is well below the epidemic threshold (Figure \ref{ABC}).
Poor capacity of MERS-CoV to spread human-to-human explains why evidence of substantial community transmission has been lacking, as well as the absence of virus circulation outside of Arabian peninsula.
Perhaps a clearer indicator of this has been the lack of a global MERS pandemic, which would have certainly taken place had R$_{0}$ been above the epidemic threshold.

Despite higher resolution of inference offered by structured population models, especially when sequence data are plentiful, serious drawbacks exist.
As the representation of evolution drifts away from a strictly bifurcating tree model, the space over which phylogenetic inference needs to take place becomes increasingly complex.
Convergence difficulties are well known in rich models, such as ancestral recombination graphs, and multitype trees are no different.
Even with a moderate number of sequences (N=274) and the smallest possible number of demes (N=2) structured population model mixing is slow and problematic.
Some convergence issues (Figure \ref{skygrid}) could be attributed to the presence of recombination in MERS-CoV data (Figure \ref{recombination}), but active research into approximations to the structured coalescent \citep{maio_new_2015} seem to acknowledge both the utility and difficulties in applying such methods.
As computational infrastructure improves \citep{ayres_beagle:_2012}, it is hopefully only a matter of time before phylogenetic population structure models mature to the point where large datasets with multiple demes (\textit{e.g.} West African Ebola virus epidemic \citep{dudas_virus_2017}, global influenza virus circulation \citep{bedford_global_2015}) can be analysed trivially.


% Similar approaches have been used to quantify population structure in American Zika viruses, where repeated introductions into Florida observed in sequence data were used to extrapolate probable numbers of viral introductions


% Population structure is often inferred via phylogeny labelling approaches where discrete states are assigned to taxa, and ancestral states are reconstructed via a continuous time Markov chain the same way nucleotide sites would be.
% These methods, when used to infer ancestral migration history across a phylogeny, have thus been called ``mugration'' models.
% When the number of possible discrete states is limited, data are ample and questions are formulated well, phylogeny labelling approaches are computationally efficient, accurate and highly informative.
% However, ``mugration'' models ignore potentially relevant information, such as markedly different demographic histories in distinct populations.
%
% Sequencing of MERS-CoV has been a challenging issue, especially from the reservoir.
% MERS-CoV seroprevalence in camels, as expected of a reservoir, is very high, yet sequencing efforts have overwhelmingly focused on human infections.
% At present MERS-CoV sequences collected from humans outnumber those from camels at a ratio of over 2-to-1, and sequences from camels appear late in the MERS epidemic, largely because of the time taken to identify camels as the reservoir.
% Hence, using MERS-CoV sequence data in ``mugration'' models have led to contradictory findings, such as inference of MERS-CoV as a repeated epizootic in camels of largely human origins \citep{zhang_evolutionary_2016}.
% Traditional epidemiological approaches fare no better, since links between human cases outside of known family clusters or hospital outbreaks are exceedingly difficult to establish \citep{cauchemez_unraveling_2016}.
%
% Our study has shown how significantly different demographic regimes between hosts such as those observed in MERS-CoV can be leveraged against biased sequence data to arrive at conclusions perfectly in line with prior observations.
% MERS-CoV sequences collected from camels are, on average, relatively distantly related to each other, whilst human outbreaks fall into densely sampled clusters of no apparent relation to each other and of no consequence to further evolution of the virus.
% These hallmarks of distinct demographic modes were easily picked up by the explicit population structure model we employ, and reveals a cohesive picture of MERS-CoV epidemiology in agreement with previous empirical studies.
% However, we do note that at present phylogenetic models with population structure are limited in terms of computational efficiency than current models, even when approximations are used.

% \subsection*{Issues with recombination}
% Recombination in MERS-CoV is now an accepted feature of the system.
% Prevalent recombination can seriously impede phylogenetic and other sequence analysis techniques.
% We argue that although MERS-CoV sequences contain overwhelming evidence for recombination, we do not expect humans to play a major role in this process.
% As human infection with MERS-CoV remains very low, every introduction of the virus into a human population is expected to establish a clonally evolving and short-lived population.
% However, lineages crossing the species barrier are very likely to have experienced recombination at some point in their past, in some cases occurring right before zoonotic transmission, as was the case in the Korean outbreak of MERS.
% We thus conclude that whilst sequence analyses of MERS-CoV are perfectly legitimate for well-defined human outbreaks, extreme care must be taken when inferring their origins in the context of the reservoir, or even when comparing related outbreaks in humans.
% We also strongly advise against drawing hasty conclusions from MERS-CoV phylogenies recovered from human sequences if they span more than four months of evolution or camel sequences under virtually all circumstances, as there is no guarantee that phylogenies will cover the clonal evolutionary history of the virus in humans.

\subsection*{Sequence data as a universal epidemiology tool}
As the field of epidemiology enters the age of genomics sequence data are likely to be brought to the forefront of epidemiological investigations.
At a relatively low price, sequence data offer the combined benefits of diagnostics and typing, whilst also offering insight into a pathogen's evolutionary history.
With a small amount of metadata, primarily collection dates and features of interest, such as location or host, much can be revealed through phylogenetic clustering of sequences, or absence thereof.
This was used to great effect towards the end of the Ebola virus epidemic in West Africa, where the final transmission chains were tracked and confirmed as such with sequence data \citep{arias_rapid_2016}.
Although sequence data can be collected for multiple reasons and used to address different questions, sequence data are a fundamental unit of ground truth, which allows data from wildly different studies to be combined trivially\lmc{TRIVIALLY???}.
In fact it has often been the combination of data from distinct populations and different time periods that yielded most insight in the past \citep{dudas_virus_2017}.

\newpage

\section*{Methods}
\subsection*{Sequence data}
All MERS-CoV sequences were downloaded from GenBank.\lmc{accession numbers are given in Table X}.
Fragments of some strains submitted to GenBank as separate accessions were assembled into a single sequence.
Protein coding sequences were extracted and concatenated.
Sequences were annotated with available collection dates and hosts, designated as camel or human.
The final dataset consisted of 174 genomes from human infections and 100 genomes from camel infections.

\subsection*{Structured coalescent analyses}

MultiTypeTree module \citep{vaughan_efficient_2014} was used in BEAUti v2.4.3 \citep{bouckaert_beast_2014} to specify a structured coalescent model with two demes - humans and camels - based on GenBank records.
Analyses were run on codon position partitioned data with two separate HKY+$\Gamma_{4}$ \citep{hky_1985,yang_1994} nucleotide substitution models specified for codon positions 1+2 and 3.
A relaxed molecular clock with branch rates drawn from a lognormal distribution \citep{drummond_2006} was used to infer the evolutionary rate from date calibrated tips.
Default priors were used for all parameters except for migration rates between demes for which an exponential prior with mean 1.0 was used.
All analyses were run across ten independent Markov chains (MCMC runs).
To confirm that demographic inference was derived from the data rather than the prior we set up an additional ten independent analyses where the mean of the exponential distribution prior for migration rate was set to 10.0.

We also analysed a dataset where alignments split into two fragments (fragment 1 comprised of positions 1-21000, fragment 2 of positions 21000-onwards), with independent clocks, trees and migration rates, but shared substitution models and deme population sizes.
Fragment positions were chosen based on consistent identification of the region around nucleotide 21000 as a probable breakpoint by GARD \citep{pond_gard:_2006} by previous studies into SARS and MERS coronaviruses \citep{hon_evidence_2008,dudas_mers-cov_2016}.
All analyses were set to run for 200 million states, subsampling every 20 000 states.
Due to the increased complexity of multitype tree parameter space 50\% of states from every analysis were discarded as burnin for single fragment analyses and 20\% for two-fragment analyses.
Chains not converging after 200 million states were discarded.
For single fragment analysis with migration rate prior set to exponential with mean 1.0 there were three chains out of ten that did not converge and two for the analysis where the migration rate prior was an exponential with mean 10.0.
For the two fragment structured coalescent analyses three chains out of ten failed to converge.

\subsection*{Demographic inference of MERS-CoV in the reservoir}
In order to infer the demographic history of MERS-CoV in camels we used the results of structured coalescent analyses to identify introductions of the virus into humans.
The oldest sequence from each cluster introduced into humans was used to represent a random draw from the diversity of MERS-CoV circulating in camels.
These sequences were combined with existing sequence data from camels to give us a dataset with minimal demographic signal coming from epidemiological processes in humans.
Sequences belonging to the outgroup clade where most of MERS-CoV sequences from Egypt fall were removed out of concern that MERS epidemics in the Arabian Peninsula and Egypt are distinct epidemics with relatively poor sampling in the latter.
A flexible skygrid tree prior \citep{gill_2013} was used to recover estimates of relative genetic diversity (N$_{e}\tau$) at 50 evenly spaced grid points across six years, ending at the most recent tip in the tree (2015 August) in BEAST v1.8.4 \citep{drummond_bayesian_2012}.
We set up five independent MCMC chains to run for 500 million states, sampling every 50 000 states.
This analysis suffered from poor convergence, where two chains converged onto one stationary distribution, two to another and the last chain onto a third stationary distribution, with high effective sample sizes.\lmc{already reported above. Why not remove?}

\subsection*{Epidemiological analyses}

As of 10 May 2017, the World Health Organization has been notified of 1952 cases of MERS-CoV.
We thus simulate final transmission chain sizes using equation \ref{clusterLikelihood} \citep{lloyd-smith_superspreading_2005,blumberg_inference_2013} until we reach an epidemic comprised of 2000 cases.
10 000 simulations were run for 121 uniformly spaced values of R$_{0}$ across the range [0.5--1.1] with dispersion parameter $\omega$ fixed to 0.1.
Each simulation results in a vector of outbreak sizes $C$, where $C_{i}$ is the size of the \textit{i}th transmission cluster and $\sum_{i=1}^{K} C_{i} = 2000$, where $K$ is the number of clusters generated.
We sample from the case cluster size vector according to a multivariate hypergeometric distribution to simulate sequencing.
The resulting sequence cluster size vector $S$ contains $K$ entries, some of which are zero (\textit{i.e.} case clusters not sequenced), but $\sum_{i=1}^{K} S_{i} = 174$ which reflects the number of MERS-CoV sequences used in this study.
Since the sampling scheme operates under equi-probable sequencing we also simulated biased sequencing by using concentrated hypergeometric distributions where the probability mass function is squared (bias = 2) or cubed (bias = 3) and then normalized.
This makes clusters likely to be `sequenced' even more likely to be sequenced and \textit{vice versa}.
We performed a smaller set of simulations with 2500 replicates and twice the number of cases, \textit{i.e.} $\sum_{i=1}^{K} C_{i} = 4000$, to explore a dramatically underreported epidemic.

Let $R_{0}$ be the basic reproductive number ($ 0 < R_{0} < 1$),  and $\omega > 0$ be the a dispersion parameter, then the probability of observing a stuttering chain of size (cluster) $r$ of size $j$ is~\citep{blumberg_inference_2013}
\begin{equation}
Pr(r = j | R_{0}, \omega) = \frac{\Gamma(\omega j+j-1)}{\Gamma(\omega j)\Gamma(j+1)} \frac{(\frac{R_{0}}{\omega})^{j-1}}{(1+\frac{R_{0}}{\omega})^{\omega j+j-1}}.
\label{clusterLikelihood}
\end{equation}

Although case clusters and their sizes are difficult to infer directly and require detailed epidemiological follow up, sequence data have fewer limitations.\lmc{other than the ``obvious'' limitation that they're \textbf{not} the actual infection process, uh? >.>}
Our structured coalescent analyses indicate that every MERS outbreak is a contained cross-species transmission of the virus from camels into humans.
The distribution of the number of these cross-species transmissions and their sizes thus contain information about the underlying distribution of case clusters.
We employ a Monte Carlo simulation approach to identify simulations where the recovered distribution of sequence cluster sizes fall within the 95\% highest posterior density intervals for three summary statistics of MERS-CoV sequence cluster sizes recovered via structured coalescent analyses: mean, median and standard deviation.
These values are 2.90--3.70 for mean sequence cluster size, 4.87--6.07 for standard deviation of sequence cluster sizes and a median sequence cluster size of 1.

\subsubsection*{Introduction seasonality}

We extracted the times of camel-to-human introductions from the posterior distribution of multitype trees.
This distribution of introduction times was then discretised as follows: for state  $k = 1, 2, \ldots, L$ in the chain,  $Z_{ijk}$ was $1$ if there as an introduction in month $i$ and year $j$ and $0$ otherwise.
We model the variable $Y_{ij} = \sum_{k = 1}^L Z_{ijk}$ with the hierarchical model:
\begin{align*}
  Y_{ij} &\sim \text{Binomial}(L, \theta_{ij}); \\
  logit(\theta_{ij}) &= \alpha_j + \beta_i; \\
  \beta_i &\sim \text{Normal}(0, \sigma_{m}); \\
  \sigma_{m} &\sim \text{Cauchy}(0, 2.5); \\
  \alpha_j &\sim \text{Normal}(\mu_{y}, \sigma_{y}); \\
  \mu_{y}  &\sim  \text{Normal}(0, 1); \\
  \sigma_{y} &\sim \text{Cauchy}(0, 2.5).
\end{align*}

Odds ratios of introductions can then be computed for each month $i$ as $\text{OR}_i = \exp(\beta_i)$.

\subsection*{Data availability}
Sequence data and all analytical code is publicly available at https://github.com/blab/structured-mers.

\section*{Acknowledgements}
We would like to thank Allison Black for useful discussion and advice.
GD is supported by the Mahan postdoctoral fellowship from the Fred Hutchinson Cancer Research Center.


\bibliographystyle{mbe.bst}
\bibliography{mers-structure}

\setcounter{figure}{0}
\setcounter{table}{0}
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\thetable}{S\arabic{table}}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_exploded.png}
	\caption{\textbf{Evolutionary history of MERS-CoV partitioned between camels and humans.}
Same tree as shown in Figure \ref{mcc}, but all contiguous stretches of MERS-CoV evolutionary history split by inferred host: camels (top) and humans (bottom).
This visualisation highlights the ephemeral nature of MERS-CoV outbreaks in humans, compared to continuous circulation of the virus in camels.
	}
	\label{exploded}
\end{figure}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_prior.png}
	\caption{\textbf{Result are robust to prior choice.}
Negligible flow of MERS-CoV lineages from humans into camels is recovered regardless of prior choice.
Plots show the 95\% highest posterior density for the estimated migration rate from the human deme into the camel deme (blue) and \textit{vice versa} (orange).
Dotted lines indicate exponential priors specified for migration rates, with mean 1.0 (bottom) or 10.0 (top).
%\tbc{I think this would be more clear if the left and right sides shared the same x-axis. As it stands it looks at first glance like there's different priors for camel-to-human compared to human-to-camel. It's okay if the blue human-to-camel rates gets smushed up against the left of the plot.}
	}
	\label{prior}
\end{figure}


\begin{figure}[h]
\centering
	\includegraphics[scale=0.25]{figures/mers_method.pdf}
	\caption{\textbf{Monte Carlo simulation method.}
Case clusters are simulated according to \ref{clusterLikelihood} until an outbreak size of 2000 cases is reached.
We sample 174 cases from each simulation to represent sequencing of human MERS cases.\lmc{is this implicitly a binomial sampling with $p = 1/2$?} 
`Sequencing' is carried out by using multivariate hypergeometric sampling, where at each iteration case cluster size $C_{i}$ is the number of successes in the population, $\sum_{i=1}^{K} C_{i} - C_{i}$ is the number of failures and number of trials is equal to leftover sequencing capacity, that is $\sum_{i=1}^{K} S_{i}$ minus whatever has been `sequenced' up to that iteration.
Sequencing simulations take place at three levels of bias - 1.0, where every case is equally likely to be sequenced, and 2.0 and 3.0, where cases from larger clusters are increasingly more likely to be sequenced.
Simulated sequence clusters are summarised by their mean and standard deviation, both of which reflect some aspect of the partitioning of sequences into clusters.
A simulation is considered to match if its sequence cluster size mean and standard deviation fell within the 95\% highest posterior density interval of observed MERS-CoV sequence clusters.
$R_{0}$ values that ultimately generate data matching empirical observations, as well as associated numbers of `introductions' are kept as estimates.
These estimates are summarised in Figure \ref{ABC}.
	}
	\label{method}
\end{figure}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_epi_4000c.png}
	\caption{\textbf{Results of Monte Carlo simulations with vast underestimation of cases.}
The plot is identical to Figure \ref{ABC}, but instead of 2000 cases,
% the currently reported number of MERS-CoV cases,
simulations were run with 4000 cases.
With more unobserved cases the R$_{0}$ values matching observed MERS-CoV sequence clusters can only be smaller, with a corresponding increase in numbers of zoonotic transmissions.
However, the numbers of simulations that match MERS-CoV data go down as well.
	}
	\label{extra_cases}
\end{figure}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_recombination.png}
	\caption{\textbf{Tests of recombination across MERS-CoV clades.}}
Maximum clade credibility tree of MERS-CoV genomes annotated with results of two recombination detection tests (PHI and 3Seq) applied to descendent sequences of each clade.
Both tests identify large portions of existing sequence data as containing signals of recombination.
Note that markings do not indicate where recombinations have occurred on the tree, merely the minimum distance in sequence/time space between recombining lineages.
	\label{recombination_tree}
\end{figure}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_incompatibilities.png}
	\caption{\textbf{MERS-CoV genomes exhibit high numbers of non-clonal loci.}
Ancestral state reconstruction (right) identifies a large number of loci that have occurred more than once in the tree (homoplasies, orange) or are reversions (red) from a state arising in an ancestor.
Mutations that apparently only occur once in the tree (synapomorphies) are shown in grey.
The maximum likelihood phylogeny on the left is coloured by whether sequences were sampled in humans (blue) or camels (orange).
	}
	\label{incompatibilities}
\end{figure}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_flower.png}
	\caption{\textbf{Human clade sharing between genomic fragments 1 and 2.}
Central scatter plot shows the posterior probability of human clades shared between genomic fragments 1 and 2, in their respective trees.
Left and bottom scatter plots track the posterior probability of human clades only observed in fragment 2 (left) or fragment 1 (bottom).
The cumulative probability of human clades present in either tree are tracked by plots on the right (fragment 2) and top (fragment 1).
Most of the probability mass is concentrated within human clades that are present in trees of both genomic fragment 1 and 2 (0.9701 and 0.9474 of all human clades across posteriors, respectively).
	}
	\label{flower}
\end{figure}

\begin{figure}[h]
\centering
	\includegraphics[width=0.65\textwidth]{figures/mers_skygrid.png}
	\caption{\textbf{Demographic history of MERS-CoV in Arabian peninsula camels.}
Demographic history of MERS-CoV in camels, as inferred via a skygrid coalescent tree prior \citep{gill_2013}.
Two skygrid reconstructions are shown, one for each of the stationary distributions reached by MCMC.
Shaded interval indicates the 95\% highest posterior density interval for the product of generation time and effective population size, $N_{e}\tau$.
Midline tracks the inferred median of $N_{e}\tau$.
Maximum clade credibility (MCC) tree from the skygrid was inferred is shown in the background, with camel sequences highlighted in orange and human sequences highlighted in blue.
	}
	\label{skygrid}
\end{figure}

\end{document}
